.idea/checkstyle-idea.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file from the commit. Add values to .gitignore to avoid commit files this